grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.html

```diff
@@ -6,12 +6,22 @@ <h2>DESCRIPTION</h2>
 
 <p>
 By default <em>i.sentinel.import</em> imports all Sentinel scene files found
-in the <b>input</b> directory by <em><a href="r.import.html">r.import</a></em>.
+in the <b>input</b> directory. The number of scene files can be optionally 
+reduced by the <b>pattern_file</b> option. In this option, a regular expression 
+for filtering the file names should be given, e.g. "MSIL2A.*T32VNR_2019" for 
+importing only level 2A products for tile T32VNR from 2019.
+
+<p>
 Note that in the case that spatial reference system of input data differs
-from GRASS GIS location, the input data are reprojected. To speed up this
+from GRASS GIS location, the input data need to be reprojected with 
+<em><a href="r.import.html">r.import</a></em>. To speed up this
 process, a higher than default value can be specified for the <b>memory</b>
 option.
 
+<p>
+In order to ignore insignificant mismatch of the spatial reference 
+systems, reprojection can be supressed with the <b>-o</b> flag.
```

Comment:
```suggestion
systems, the projection check can be suppressed with the <b>-o</b> flag.
```
... as -o does that.